src/mailadm/conn.py

```diff
@@ -232,6 +232,49 @@ def get_expired_users(self, sysdate):
             users.append(UserInfo(*args))
         return users
 
+    def get_users_to_warn(self, sysdate: int) -> [{}]:
+        q = UserInfo._select_user_columns
+        allusers = []
+        users_to_warn = []
+        for args in self._sqlconn.execute(q, (sysdate, )).fetchall():
+            allusers.append(UserInfo(*args))
+        for user in allusers:
+            year = 31536000
+            month = 2592000
+            week = 2592000
+            day = 86400
+            warnmsg = """Your account will expire in ?. You should look for an alternative email
+                provider right now. With Delta Chat, you can keep all your chats and conversations;
+                just use the AEAP mechanism to tell your contacts of your address migration: <link>
+
+                Your %s team""" % (self.config.mail_domain,)
```

Comment:
This was just a first draft for the warning; a link where AEAP is explained nicely doesn't exist yet.

Reply:
Can we link to the Delta Chat in-app-help?

Reply:
I linked to https://delta.chat/en/2022-09-14-aeap for now. There should be an FAQ item for this in the long term.